.github/actions/build-sdist/action.yml

```diff
@@ -0,0 +1,62 @@
+name: 'Build Source Distribution'
+description: 'Build a source distribution for an ofrak package'
+inputs:
+  package-name:
+    description: 'Name of the package to build'
+    required: true
+  source-dir:
+    description: 'Source directory containing the package'
+    required: true
+  lfs:
+    description: 'Whether this package requires LFS'
+    required: false
+    default: 'false'
+  python-version:
+    description: 'Python version to use for building'
+    required: true
+runs:
+  using: 'composite'
+  steps:
+    - name: Checkout code
+      uses: actions/checkout@v4
+      with:
+        fetch-depth: 0
+        lfs: false
+
+    - name: Setup LFS cache
+      if: ${{ inputs.lfs == 'true' }}
+      uses: ./.github/actions/lfs-cache
+
+    - name: Set up Python
+      uses: actions/setup-python@v4
+      with:
+        python-version: ${{ inputs.python-version }}
+        cache: pip
+        cache-dependency-path: |
+          requirements-*.txt
+          **/setup.py
+
+    - name: Install build tools
+      shell: bash -euxo pipefail {0}
```

Comment:
Why set this for each step instead of as a default at the top? Does that not work here?